Makefile

```diff
@@ -635,6 +635,10 @@ package-capi:
 		cp target/headless/$(CARGO_TARGET)/release/wasmer.dll package/lib/wasmer-headless.dll ;\
 	fi
 
+	if [ -f target/headless/$(HOST_TARGET)/release/wasmer.dll ]; then \
```

Comment:
I assume the duplication of these copies is intentional to cover both native and cross compilation?

Reply:
yes, but it's to cover CI vs Local also, because local use cross-compile to HOST_TARGET (or else the build would not work for some obscure reasons)